### ISSUE TITLE
Feature request: include items in "people" tab

### ISSUE THREAD

**Issue by ccrowles** [Marmo/debitum#122]
As per title, currently in the People tab it says "no debt" if someone only owes you items.
![Screenshot_20221125-141253_Debitum](https://user-images.githubusercontent.com/12866474/203994459-f968fb5e-aac8-462e-b2e0-f4545aa740e8.jpg)

![Screenshot_20221125-141250_Debitum](https://user-images.githubusercontent.com/12866474/203994389-f6a4c495-c7f4-469e-a7a1-fef6750d6538.jpg)

It would be nice to have something that includes items, i.e:
![IMG_20221125_141847](https://user-images.githubusercontent.com/12866474/203994323-f6f4f21e-8477-4731-a967-233575df5198.png)

**Comment by Marmo** [comment]
Good point, just need to define how items are aggregated (summing up like with money would make no sense). Best would be imo summing the absolute values (I gave them one book + I got a hammer from them should show up as "2 items" not "0 items")

**Comment by ccrowles** [comment]
Yeah, I think your idea would probably be the best way for the total.
